treeherder/seta/management/commands/reset_seta.py

```diff
@@ -1,13 +1,18 @@
 from django.core.management.base import BaseCommand
```

Comment:
I think @edmorley 's summary is accurate. I will have to look at this either later today or Monday.